### ISSUE TITLE
Autosquash doesn't squash on PRs with a single commit

### ISSUE THREAD

**Issue by kilianc** [tibdex/autosquash#34]
I don't think this is ideal given your comment: https://github.com/tibdex/autosquash/issues/24#issuecomment-557820239

I would like my team to only focus on PR title and description - I am also enforcing semantic PR title for a clean commit history.

**Comment by tibdex** [comment]
I've seen PRs with a single commit get merged by Autosquash and the merged commit's message was the PR description as expected.

**Comment by kilianc** [comment]
I'll record a quick video that showcases the problem. It is currently not. In fact, I had to force push our develop branch because our changelog scripts were broken, 3 times in the past 2 months.

**Comment by Kineolyan** [comment]
Hi @tibdex, I wanted to have a look at this, but I wasn't sure how to process. Can you briefly describe your test process here? I can do a PR after that to include it in the Readme

**Comment by tibdex** [comment]
Well this issue seems to report a problem where Autosquash would merge PRs made of a single commit but the message of the resulting commit on the base branch wouldn’t be the description of the PR as expected but I guess the message of the initial commit in the head branch. 
I haven’t seen that happening myself so a first step would be to give a link to a PR where this occurred.

**Comment by kilianc** [comment]
I can't share the repo becuase it's private but I found an example where I had a typo in my commit and PR title. I fixed the PR title thinking that `autosquash` would discard the commit and use the PR title and first comment instead.

<img width="778" alt="Screen Shot 2020-04-12 at 2 03 28 AM" src="https://user-images.githubusercontent.com/57916611/79064989-281f4a80-7c62-11ea-9e57-4c1fce2f347e.png">
<img width="1023" alt="Screen Shot 2020-04-12 at 2 03 38 AM" src="https://user-images.githubusercontent.com/57916611/79064996-2c4b6800-7c62-11ea-8b33-5f51fac2bb44.png">
<img width="1004" alt="Screen Shot 2020-04-12 at 2 04 03 AM" src="https://user-images.githubusercontent.com/57916611/79064997-2f465880-7c62-11ea-9512-2f037aa7a056.png">

Notice I fixed the `update` vs `updated` in the PR but the bot used the commit message instead.
